8_distributed_training.ipynb

```diff
@@ -72,13 +72,22 @@
     "\n",
     "2. Click *Display token*.\n",
     "\n",
-    "3. Below *Log in with this token*, copy the line that has the following format:\n",
-    "\n",
-    "    `oc login --token=sha256~XXXX --server=https://XXXX`\n",
-    "\n",
-    "    You can run the `oc login` command the same way that you run any terminal command. You can designate a cell as `%%bash` or you can paste a command after `!`.\n",
-    "\n",
-    "4. Paste your `oc login --token=sha256~XXXX --server=https://XXXX` command in the following cell designated with `%%bash` and run it.\n"
+    "3. Below *Log in with this token*, take note of the parameters for token and server.  \n",
```

Comment:
change  *Log in with this token* to bold text > **Log in with this token** (make similar change in steps 1 and 2)